test/NinjaTools.FluentMockServer.API.Tests/Models/Logging/LogItemTests.cs

```diff
@@ -0,0 +1,170 @@
+using FluentAssertions;
+using JsonPrettyPrinterPlus;
+using Microsoft.AspNetCore.Http;
+using NinjaTools.FluentMockServer.API.Helper;
+using NinjaTools.FluentMockServer.API.Models;
+using NinjaTools.FluentMockServer.API.Models.Logging;
+using NinjaTools.FluentMockServer.Tests.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+using HttpResponse = NinjaTools.FluentMockServer.API.Models.HttpResponse;
+
+namespace NinjaTools.FluentMockServer.API.Tests.Models.Logging
+{
+    public class LogItemTestData : TheoryData<ILogItem,  string>
+    {
+        public static HttpContext Context = new DefaultHttpContext()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802642)